sdk/search/Azure.Search.Documents/tests/Utilities/SearchRecordedTestSanitizer.cs

```diff
@@ -61,26 +79,5 @@ public override void SanitizeHeaders(IDictionary<string, string[]> headers)
             }
             base.SanitizeHeaders(headers);
```

Comment:
This now needs to be handled as well since the base class is now just a go-between. I can make similar changes, but it means duplicating everything the base class is doing.

Reply:
We still call into the base method https://github.com/Azure/azure-sdk-for-net/pull/11582/files#diff-c7bc11e4c6e4461a87aa7a15a4a2aa87R49  so SanitizeHeaders would get called from there.

Reply:
I can only do that for headers. Everything else I have to duplicate. Hardly seems worth it compared to switching order to something that worked before, but I'm making the changes in my PR because this change is not enough to sanitize all the right content.

Reply:
Yes, I'm doing that in my changes. But it has to be done unconditionally, which is another reason this change won't work. Because I'm overriding the main entry point, I have to duplicate a lot more to make sure everything gets called correctly *and* the sanitizing the base class is doing happens. If that base class ends up doing more, this code will not stay in sync. All this duplication will eventually be out of date and is exactly what I was trying to avoid.

Reply:
Frankly, I feel like making secret variable sanitization in the headers, body, etc., is more valuable in the long run and a lot less duplication of code.